external/swe-polybench/evaluation/evaluate.py

```
@@ -0,0 +1,153 @@
+            str(run_script),
+            "--dataset-path", str(args.dataset_path.resolve() if args.dataset_path.is_file() else args.dataset_path),
+            "--predictions-path", str(args.predictions_path.resolve()),
+            "--result-path", str(args.result_path.resolve()),
+            "--num-threads", str(args.num_threads),
+            "--delete-image",
+        ]
+        if args.repo_path is not None:
+            cmd.extend(["--repo-path", str(args.repo_path.resolve())])
+        # subprocess.run uses shell=False; cwd and script path are operator-controlled.
+        try:
+            subprocess.run(cmd, cwd=str(repo_dir), check=True)
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)